frontend/src/components/punishment/PunishmentItem.tsx

```diff
@@ -91,8 +91,7 @@ export const PunishmentItem = ({
 
   const date = dayjs.utc(punishment.created_at).tz("Europe/Oslo")
 
-  const formattedDate = date.format("DD. MMM HH:mm")
-  const formattedLongerDate = date.format("DD. MMM YYYY HH:mm")
+  const formattedDate = date.format("kl. HH:mm, DD. MMM YYYY")
```

Comment:
Maybe do it so that if year is not current year, show year?

Reply:
the only issue i see with only sometimes showing the year is that since it is right aligned it becomes quite ugly

ex.

```
     kl. 12:34, 14. mai
kl. 12:34, 14. des 2023
```

Reply:
daså